### DISCUSSION TITLE
Button.Group should use inline-flex instead of flex

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#4740] [Feedback]
<img width="349" alt="image" src="https://github.com/mantinedev/mantine/assets/124666577/bdc8eef9-800e-4f12-8d11-428c544dc46b">

**Comment by 12rambau** [comment]
This conversation have been started some time ago with no traction but i'm actually facing the same problem. To describe it with more words, my problem is that the `ButtonGroup` is not merging the borders where 2 buttons are touching making it either very think when the colors are the same (my case) or clearly divergent when like in the OP message, they should blend. 

<img width="356" height="46" alt="image" src="https://github.com/user-attachments/assets/7a72f04d-4f95-4257-9956-9340dc38f830" />


Would it be possible to change that ? 

Note: I'm not using `mantine` directly but it's port to Dash for python users, I can eventually try to work on it but I'm not a seasoned front-end developer and will thus require some guidances to understand where to look. 

I check your documentation to make sure my question is aligned with latest and it seems it's the case according to what i see in https://mantine.dev/core/button/#buttongroup

**Reply by rtivital** [reply]
I see that in that demo borders are merged – there is 1px border between buttons. Can you check with the team who suppors dash components to see if they've missed something? 

<img width="398" height="157" alt="Screenshot 2026-03-11 at 17 42 18" src="https://github.com/user-attachments/assets/a5ec415d-001d-4b67-b5ba-3efc689aab9e" />

**Comment by 12rambau** [comment]
this is what I see when I load the demo page and they don't look merged to me: 

<img width="479" height="161" alt="image" src="https://github.com/user-attachments/assets/63766447-8193-475f-aee9-2baea9bbe74a" />

But if I zoom at 200% they get merged back: 

<img width="1020" height="281" alt="image" src="https://github.com/user-attachments/assets/417d35d8-4c72-4479-b656-c3d4fc7ed246" />

I have the same behaviour on chrome, edge and firefox, is it device related ?
